src/par_iter/splitter.rs

```diff
@@ -0,0 +1,66 @@
+use super::internal::*;
+use super::*;
+
+/// The `split` function takes arbitrary data and a closure that knows how to
+/// split it, and turns this into a `ParallelIterator`.
```

Comment:
(I'm not sure of what the "unstable" *policy* is -- probably somewhat we should chat about. Maybe rather than a cargo feature, just some doc comments might be good.)